src/main/java/com/google/api/codegen/GeneratorBuilderUtil.java

```diff
@@ -14,13 +14,6 @@
  */
```

Comment:
The class doc comments here are no longer accurate. (They reference a "codegen provider".)